readme.md

```diff
@@ -73,18 +73,18 @@ const redirectURL = 'http://mypatreonapp.com/oauth/redirect'
 function handleOAuthRedirectRequest(request, response) {
     const oauthGrantCode = url.parse(request.url, true).query.code
 
-    patreonOAuthClient.getTokens(oauthGrantCode, redirectURL, (tokensError, { access_token }) => {
-        const patreonAPIClient = patreonAPI(access_token)
-
-        patreonAPIClient(`/current_user`, (currentUserError, apiResponse) => {
-            if (currentUserError) {
-                console.error(currentUserError)
-                response.end(currentUserError)
-            }
-
-            response.end(apiResponse);
+    patreonOAuthClient.getTokens(oauthGrantCode, redirectURL)
```

Comment:
`getToken` w/ no s?

Reply:
yea i changed it back to `getTokens`, because it returns more than one token